README.md

```diff
@@ -345,6 +345,7 @@ RUNTIME_IGNOREDOTFILES default:"false"
 There are two methods for triggering a configuration reload:
 1. Symlink RUNTIME_ROOT to a different directory.
 2. Update the contents inside `RUNTIME_ROOT/RUNTIME_SUBDIRECTORY/config/` directly.
+3. Add/Update the existing domain descriptors `UPSERT_DESCRIPTORS` default `false`
```

Comment:
What does `UPSERT_DESCRIPTORS` mean? Can you describe what this does and/or have a better name?

Reply:
@mattklein123  Could you suggest a better name?  I am actually looking for feature where apps can add/delete descriptors dynamically .

Reply:
@mattklein123  integration tests are failing , I think its from latest merge. Could you help me in resolving that ?

Reply:
Main branch is broken right now, see https://github.com/envoyproxy/ratelimit/pull/213.

I still don't understand what this change does. Can you update the description to remove Envoy/Istio and just be clear about what we are trying to do with the rate limit service itself? Thank you.

/wait

Reply:
@mattklein123  Added few lines , I basically want to update the existing domain descriptiors using config files.(RUNTIME_WATCH_ROOT = false)

Reply:
Sorry what lines did you add? Please fully update the description to remove Istio/Envoy and talk about this change in isolation in terms of what it does for the ratelimit service. Thank you!

/wait